tests/node/fix1839/run

```diff
@@ -0,0 +1,4 @@
+rm -rf build
+
+$1 --cg node --no-banner --no-color --console-width 0 OS.idr < input
+$1 --cg javascript --no-color -o os.js OS.idr
```

Comment:
You're missing `--no-banner` and `--console-width 0` here, so maybe it's failing due to the console on windows being a different width.

Reply:
Ah, yes, that makes sense. Thanks for spotting this!